prefect_azure/deployments/steps.py

```diff
@@ -0,0 +1,196 @@
+"""
+Prefect deployment steps for code storage and retrieval in Azure Blob Storage.
+
+These steps can be used in a `prefect.yaml` file to define the default 
+push and pull steps for a group of deployments, or they can be used to 
+define the push and pull steps for a specific deployment.
+
+!!! example
+    Sample `prefect.yaml` file that is configured to push and pull to and 
+    from an Azure Blob Storage container:
+
+    ```yaml
+    prefect_version: ...
+    name: ...
+
+    push:
+        - prefect_azure.deployments.steps.push_to_azure_blob_storage:
+            requires: prefect-azure[blob_storage]
+            container: my-container
+            folder: my-project
```

Comment:
not sure if you want to call it something different than `my-project` to differentiate from "projects" at all 🤷‍♂️